diff --git a/dlls/win32u/vulkan.c b/dlls/win32u/vulkan.c
index 49eac27..8d6f537 100644
--- a/dlls/win32u/vulkan.c
+++ b/dlls/win32u/vulkan.c
@@ -3006,7 +3006,11 @@ static void vulkan_init_once(void)
     if (!(libvulkan = getenv( "CX_LIBVULKAN" )) ||
         !(graphics_backend = getenv( "CX_ACTIVE_GRAPHICS_BACKEND" )) ||
         strcmp( graphics_backend, "wined3d" ))
+#ifdef SONAME_LIBVULKAN
         libvulkan = SONAME_LIBVULKAN;
+#else
+        libvulkan = NULL;
+#endif
     else
         ERR( "Using %s\n", libvulkan );
 
